x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/create_analytics_advanced_editor/create_analytics_advanced_editor.tsx

```diff
@@ -42,11 +45,29 @@ export const CreateAnalyticsAdvancedEditor: FC<CreateAnalyticsFormProps> = (prop
   } = state.form;
 
   const forceInput = useRef<HTMLInputElement | null>(null);
+  const { toasts } = useNotifications();
 
   const onChange = (str: string) => {
     setAdvancedEditorRawString(str);
   };
 
+  const debouncedJobIdCheck = debounce(async () => {
```

Comment:
looks like these checks should be memoized to ensure the cancel call later works correctly

Reply:
Updated in 93c9f466658f2d5ec43dd9986833649c553f86f5